### PR TITLE
addded: rating and toast components

### PR DESCRIPTION
**Pull request by rkapoor10**
- Toast 
toast

- Rating
rating

### CODE REVIEW

File: components/alert/alert.html [hunk 1]
```diff
@@ -124,35 +124,36 @@ <h2 class="mb-1">Outline - Alerts</h2>
...
                             </div>
                           </div>
-                    </div>
+                   
                     
                 </div>
                 <div class="component-code">
                     <code>
-                        <iframe src="https://www.thiscodeworks.com/embed/61fc2afc6ab14d0015c32ded" style="width: 100%; height: 385px;" frameborder="0"></iframe>
+
```
Comment (Kothalkarpriya) [inline components/alert/alert.html:156]: remove the extra spaces

File: components/rating/rating.html [hunk 2]
```diff
@@ -77,6 +80,41 @@ <h2 class="mb-1">Navbar</h2>
...
+                                <i class="fas fa-star star-rating-icon"></i>
+                            </label>
+                            <label for="stars">
+                                <input class="star-rating-input" type="checkbox" >
+                                <i class="fas fa-star star-rating-icon"></i>
+                            </label>
+                        </div>
+                    </div>
+                </div>
+                <!-- <div class="component-code">
```
Comment (Kothalkarpriya) [inline components/rating/rating.html:110]: remove the comments

**Review by Kothalkarpriya — APPROVED** [review]
You can merge

File: components/alert/alert.css [hunk 3]
```diff
@@ -81,44 +81,29 @@
...
-  
-    bottom: 0; */
-    /* given to center the div */
-    /* right: 0;
-    left: 0;
-    margin: 0 auto 1rem auto; */
-    /* display:flex; */
+
     flex-direction: column;
     align-items: initial;
```
Comment (JantuDeb) [inline components/alert/alert.css:98]: I'm not sure what it does actually,

**Review by JantuDeb — APPROVED** [review]
Looks good to me

**Pull request merged**